.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     name: Ruby ${{ matrix.version }}
     strategy:
       matrix:
-        version: ['2.7', '3.0', '3.1', '3.2']
+        version: ['3.0', '3.1', '3.2']
```

Comment:
I think this means we're also dropping v2 support from this gem, which would make it a breaking change right? Should we also update the Gemfile / add a changelog entry to that effect?